pyproject.toml

```diff
@@ -51,10 +51,12 @@ dependencies = [
 ]
 
 [tool.setuptools.packages.find]
-include = ["dspy*"]
+where = ["."]
+include = ["dspy", "dspy.*"]
+exclude = ["tests", "tests.*"]
```

Comment:
curious - since we do explicit `include`, do we still need this `exclude` part? 

Non-blocking, feel free to merge!

Reply:
That's not necessary (I confirmed "tests" are not included when removing this line), but mostly for the safeguard. We do the same configuration in mlflow: https://github.com/mlflow/mlflow/blob/master/pyproject.release.toml#L139